Formula/ansible.rb

```diff
@@ -228,14 +227,19 @@ class Ansible < Formula
     sha256 "b348835825c9dcd251d9aad1f89f257277ac198a3e35a61980ab4cb28c75216b"
   end
 
+  resource "future" do
+    url "https://files.pythonhosted.org/packages/45/0b/38b06fd9b92dc2b68d58b75f900e97884c45bedd2ff83203d933cf5851c9/future-0.18.2.tar.gz"
+    sha256 "b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d"
+  end
```

Comment:
Could you please double-check that we do need to add `future` since it's a package for python2 compatibility with python3, and we have python3. 
And where is came from btw? Maybe we need to check the other added resources as well

Reply:
I will check. I don't know where it came from, it just was outputted via `poet`

Reply:
If I can figure out why `brew test ansible` doesn't work that might help too

Reply:
I figured out why it didn't work, so I'm rebuilding then I'll figure out if we need `future` and the other packages.

Reply:
This has been removed. I reviewed the other packages and they seem necessary